latch_cli/services/local_dev.py

```diff
@@ -233,22 +266,32 @@ def file_filter(filename: str) -> bool:
                                 continue
 
                             if cmd in QUIT_COMMANDS:
-                                await aioconsole.aprint(
-                                    "Exiting local development session"
-                                )
-                                await ws.close()
-                                return
-
-                            if cmd.startswith("run"):
-                                await aioconsole.aprint(
-                                    "Syncing your local changes... "
-                                )
+                                raise EOFError
+
+                            if not COMMAND_EXPRESSION.match(cmd):
+                                await aioconsole.aprint("Invalid command")
+                                continue
+
+                            if (
+                                cmd.startswith("run")
```

Comment:
you should actually be able to include arbitrary whitespace characters before your first command and still get desired behavior (try with bash)

Reply:
fixed